### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by Alepod**
Minor improvements & fixed typos

### CODE REVIEW

File: src/main/resources/assets/farmersdelight/lang/ru_ru.json [hunk 1]
```diff
@@ -153,10 +153,10 @@
 
   "farmersdelight.tooltip.cooking_pot.empty": "Пусто",
   "farmersdelight.tooltip.cooking_pot.single_serving": "Содержит одну порцию",
-  "farmersdelight.tooltip.cooking_pot.many_servings": "Содержит %s порций",
+  "farmersdelight.tooltip.cooking_pot.many_servings": "Содержит порций: %s ",
```
Comment (vectorwing) [inline src/main/resources/assets/farmersdelight/lang/ru_ru.json:156]: I think I prefer the old sentence structure. Do you feel this one is gramatically better in Russian?

**Review by vectorwing — COMMENTED** [review]
Looks good! I just made a small comment over a sentence structure you changed, but it's non-impeditive. Let me know which way is better and I'll merge. =)

**Comment by Alepod** [comment]
Old one indeed looks better, but it doesn't appear to be gramatically correct with number with last digit of 2, 3 or 4 (including actually 2, 3 and 4)

**Comment by vectorwing** [comment]
Sounds good to me, then! I'll merge this now, but feel free to make new ones if you have further updates. 👍

**Pull request merged**